src/vdom.rs

```diff
@@ -379,23 +379,40 @@ impl<Ms, Mdl, ElC: ElContainer<Ms> + 'static> App<Ms, Mdl, ElC> {
     pub fn run(self) -> Self {
         // Our initial render. Can't initialize in new due to mailbox() requiring self.
         // "new" name is for consistency with `update` function.
+        // this section parent is a placeholder, so we can iterate over children
+        // in a way consistent with patching code.
         let mut new = El::empty(dom_types::Tag::Section);
```

Comment:
Wouldn't be an artificial  `Tag` like `None` / `Placeholder` / `Empty` more suitable?

Reply:
Agreed.

Reply:
Done; Placeholder.